ambari/src/test/java/io/brooklyn/ambari/server/AmbariServerImplTest.java

```diff
@@ -49,17 +51,29 @@ public void testNullJsonThrows() {
 
     @Test
     public void testOneHostReturnsSingleItemInList() {
-        assertThat(getHostsFromJson(JSON_WITH_ONE_HOST), contains("ip-10-121-18-69.eu-west-1.compute.internal"));
+        List<String> hosts = getHostsFromJson(JSON_WITH_ONE_HOST);
+
+        assertEquals(1, hosts.size());
+        assertThat(hosts, contains("ip-10-121-18-69.eu-west-1.compute.internal"));
     }
```

Comment:
Both tests now have two reasons to fail - worth adding error messages for asserts